hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/src/test/java/org/apache/hadoop/yarn/server/applicationhistoryservice/TestApplicationHistoryManagerOnTimelineStore.java

```diff
@@ -324,7 +326,7 @@ public ContainerReport run() throws Exception {
   @Test
   public void testGetApplications() throws Exception {
     Collection<ApplicationReport> apps =
-        historyManager.getAllApplications().values();
+        historyManager.getApplications(Long.MAX_VALUE).values();
```

Comment:
No we set it to Long.MAX_VALUE to have the maximum application returned depending to the APPLICATION_HISTORY_PREFIX_MAX_APPS max app define